README.md

```diff
@@ -3,36 +3,115 @@
 ## About this project
 
 ### What is Nightfall SDK?
-Software Development Kit for interacting with Polygon Nightfall [about Polygon Nightfall](https://docs.polygon-nightfall.technology/).
+
+Software Development Kit for interacting with Polygon Nightfall.
+About [Polygon Nightfall](https://docs.polygon-nightfall.technology/).
 
 ### What is the SDK for?
 
-- Deposit your tokens from Ethereum L1 to Polygon Nightfall L2
-WIP
+- Deposit ERC20 tokens from Ethereum L1 to Polygon Nightfall L2
+- Transfer ERC20 token commitments on Polygon Nightfall L2
+- Withdraw ERC20 token commitments from Polygon Nightfall L2 to Ethereum L1
+- Check ERC20 balances on Polygon Nightfall L2
+- Safely export and import ERC20 transaction commitments from and to Polygon Nightfall L2
+- All the above applicable for ERC721 and ERC1155 token standards - WIP
 
 ## Requirements
 
+To use the Nightfall SDK you would have to be aligned with the following requirements:
+
 ### Node
+
 This project uses Node.js 16.15.1. There is an `.nvmrc` file for those using [nvm](https://github.com/nvm-sh/nvm).
 
 ### Nightfall client
-TBC
 
-### Eth testing account and tokens
-TBC
+#### What is a Nightfall Client?
 
-## Setup
+A Nightfall Client is a key part of the Nightfall architecture. The SDK uses Nightfall Client to interact with Layer 2.
```

Comment:
I think that we mentioned to avoid explaining Nightfall features in great detail, focusing on how to setup.
For example looking at the Wallet docs, there is a lot of text and examples. I had the impression we should keep it short. We can add more text for sure. 
https://docs.polygon.technology/docs/nightfall/tools/nightfall-wallet#important-information-about-transfers

Reply:
`The Nightfall Client is a key part of the architecture, it is the part  which generates Zero-Knowledge proofs, stores Zero-Knowledge commitments and interacts with the Contracts. The SDK uses Nightfall Client to interact with the Nightfall Protocol. To be able to use the SDK one must have a client running.`

Reply:
If we say something like "The Client is programmable" and you can change the code and run your own personal updated client at this point we will just confuse people.